### PR TITLE
PolySwarm Integration

### PR DESCRIPTION
**Pull request by JavierBotella**
## Status
Ready/In Progress/In Hold(Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------

## Required version of Demisto
x.x.x

## Does it break backward compatibility?
   - Yes
       - Further details:
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] Dependency 1
- [ ] Dependency 2
- [ ] Dependency 3

## Additional changes
Describe additional changes done, for example adding a function to common server.

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](link)
- [ ] [CHANGELOG](link)

### CODE REVIEW

**Comment by mrtizmoatwork** [comment]
I just now signed the CLA agreement.

File: Integrations/PolySwarm/PolySwarm.yml [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+    - contextPath: File.Type
+      description: File type e.g. "PE"
+      type: String
+    - contextPath: File MD5
+      description: MD5 hash of the file
+      type: String
+    - contextPath: File.Extension
+      description: File Extension
+      type: String
+    - contextPath: PolySwarm.FileID
```
Comment (yaakovi) [inline Integrations/PolySwarm/PolySwarm.yml]: missing description here

Comment (JavierBotella) [reply Integrations/PolySwarm/PolySwarm.yml]: done

**Comment by richardbluestone** [comment]
I've reviewed and updated the yml file.

File: Integrations/PolySwarm/Pipfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+pylint = "*"
+pytest = "*"
+mock = "*"
+pytest-mock = "*"
+requests = "*"
+requests-mock = "*"
+builtins = "*"
+
+[requires]
+python_version = "2.7"
```
Comment (anara123) [inline Integrations/PolySwarm/Pipfile]: seems that you are using python3 but in Pipfile it defined 2.7.
please update the python version

File: Integrations/PolySwarm/PolySwarm.yml [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+    - contextPath: PolySwarm.Artifact
+      description: The Artifact queried.
+      type: String
+  isfetch: false
+  longRunning: false
+  longRunningPort: false
+  runonce: false
+  script: '-'
+  type: python
+  subtype: python2
```
Comment (anara123) [inline Integrations/PolySwarm/PolySwarm.yml]: same here. Subtype should be python3

File: Integrations/PolySwarm/Pipfile (hunk 2 above)
Comment (JavierBotella) [reply Integrations/PolySwarm/Pipfile]: i'm using python2.7

File: Integrations/PolySwarm/PolySwarm.yml (hunk 3 above)
Comment (JavierBotella) [reply Integrations/PolySwarm/PolySwarm.yml]: i'm using python2.7

**Pull request merged**